RomeALincei/ANLcr2.xml

```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> አመ፡ ፲ወ፪ ለኅዳር፡ በዛቲ፡ ዕለት፡ ተዝካረ፡ በዓሉ፡
+                                        ለቅዱስ፡ ሚካኤል፡ ወርእሰ፡ ሰማያውያን፡ መሐረ፡ ለዘመደ፡ እጓለ፡ እመሕያው፡ ዘይቀውሙ፡ ኵሎ፡
+                                        ጊዜ፡ ቅድመ፡ መንበሩ፡ ለንጉሠ፡ ሰማይ፡ ወይተንብል፡ በእንቲአነ፡ ወዝ፡ ውእቱ፡ ዘርእዮ፡ ኢያሱ፡
+                                        ወልደ፡ ነዌ፡ በአምሳለ፡ ሐራሁ፡ ለንጉሥ፡ ወደንገፁ፡ እምኔሁ፡</incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.1.5">
-                                    <locus from="17vb" to="18ra"></locus>
+                                    <locus from="17vb" to="18ra"/>
                                     <title type="complete" ref="LIT1295Dersan#SalamHedar"/>
-                                    <incipit xml:lang="gez">ሰላም፡ እብል፡ ኪያከ፡ መልአክ፤</incipit>
+                                    <incipit xml:lang="gez">ሰላም፡ እብል፡ ኪያከ፡ መልአከ፤</incipit>
                                 </msItem>
                             </msItem>
+                            
                             <msItem xml:id="ms_i1.2">
-                                <locus from="18ra" to="24va"></locus>
-                                <title ref="LIT1295Dersan#Tahsas"></title>
+                                <locus from="18ra" to="24va"/>
+                                <title ref="LIT1295Dersan#Tahsas"/>
+                                
                                 <msItem xml:id="ms_i1.2.1">
-                                    <locus from="18ra" to="23va"></locus>
-                                    <title ref="LIT1295Dersan#HomilyTahsas"></title>
+                                    <locus from="18ra" to="23va"/>
+                                    <title ref="LIT1295Dersan#HomilyTahsas"/>
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ድርሳን፡ ወነገር፡ <cb n="b"/> ዘይትነበብ፡ በበዓለ፡ ቅዱስ፡
+                                        ሚካኤል፡ በወርኃ፡ ታኅሣሥ፡ ጸሎቱ፡ <gap reason="ellipsis"/> ስምዑ፡ ዘንተ፡ ኵልክሙ፡ አሕዛብ፡</incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.2.2">
-                                    <locus from="23va" to="24ra"></locus>
-                                    <title type="complete" ref="LIT1295Dersan#MiracleTerr">Miracle of a rich man reduced to poverty</title>
-                                    <incipit xml:lang="gez">
-                                        ተአምሪሁ፡  ወሀሎ፡ ፩ብእሲ፡ ክርስቲያናዊ፡ 
-                                    </incipit>                                    
-                                    <note>In <ref type="mss" corresp="BAVet82"></ref> it appears for Ṭərr</note>
+                                    <locus from="23va" to="24ra"/>
+                                    <title type="complete" ref="LIT1295Dersan#MiracleTerr">Miracle
```

Comment:
@CarstenHoffmannMarburg I do not find this miracle now at all. Please make sure that you may rename things but you may not remove things that have been already referred to multiple times. 
Please restore all deleted items.
Thank you.